internal/prefixcollector/collector_configmap_output_test.go

```diff
@@ -150,7 +152,12 @@ func (eps *ExcludedPrefixesSuite) TestKubeAdmConfigSource() {
 	eps.testCollectorWithConfigmapOutput(ctx, notifyChan, expectedResult, sources)
 }
 
+// todo - change test later
+// totally unstable test: when update comes from different sources councurrently it's not always updated,
+// because output configmap is watched for changes and restores prevous state when someone else trying to modify it
 func (eps *ExcludedPrefixesSuite) TestAllSources() {
+	eps.Suite.T().Skip("test doesn't consider concurrency")
```

Comment:
Should we consider this?

Reply:
for test to work stable we should
now it produces random failure/success results

Reply:
ok, fixed